### DISCUSSION TITLE
Error just following the Getting Started guide

### DISCUSSION THREAD

**Discussion by amesas** [avitorio/outstatic#74] [Q&A]
NextJS 12, but tried upgrading to NextJS 13 and same prob.

npm install outstatic
Create /pages/outstatic/[[...ost]].tsx
Create /pages/api/outstatic/[[...ost]].tsx
Visit http://localhost:3000

We get:
NextRouter was not mounted. https://nextjs.org/docs/messages/next-router-not-mounted
file:/node_modules/outstatic/dist/index.js (81:36661)

**Comment by avitorio** [comment]
That's odd. Can you tell me the exact version of next.js you are using?

**Reply by amesas** [reply]
Tested 13.0.5 and 12.3.3

**Reply by amesas** [reply]
I've create brand new project from zero and it works, it must be something related to out next.config or webpack.config

Let me dive into it and if it's something that could help other people I will add some feedback.

**Reply by avitorio** [reply]
Thank you!

**Reply by thecodedrift** [reply]
Looks related to the issue @ #78. Do these look like the same underlying cause?

**Reply by avitorio** [reply]
Yes, thanks for pointing it out. I've added that info to the docs.
